app/posts/publish-teacher-training-courses/2024-06-21-rollover-2024-changes.md

```diff
@@ -0,0 +1,102 @@
+4. Changes to Course Details
+5. SEND checkbox to radio buttons
+6. Full Time or Part Time radio buttons to checkboxes
+7. Add a Second Language validation
+
+### 1. Shortening Pages
+
+The Course Information page comprised of three sections - About the Course, Interview Process, and How Placements Work. Previous work had been done on the prototype to break this down into three seperate pages instead - following GDS guidance of one item per page.
+
+We relooked at this, and updated the copy and page titles following more recent research feedback.
+
+Additionaly, we moved the Markdown Guidance from the right-hand column to be in-situ above the text area so it's easier to refer too
+
+![Screenshot of course details page](about-this-course.png "Course details page")
+![Screenshot of interview process page](interview-process.png "Interview process page")
+![Screenshot of course school placement](school-placements.png "School placement page")
+
+The Course Length & Fees page comprised of four sections - Course Length, Course Fees, and Fees, and Financial Support. Previous work had been done on the prototype to break this down into four seperate pages instead - following GDS guidance of one item per page.
+
+We relooked at this, and updated the copy and page titles following more recent research feedback.
+
+We also merged the two fields/pages Fee Details and Financial Support because data showed that providers tended to enter all financial information into fee details
+
+Additionaly, we moved the Markdown Guidance from the right-hand column to be in-situ above the text area so it's easier to refer too
+
+![Screenshot of course fees page](course-fees.png "Course fees page")
+![Screenshot of course length](course-length.png "Course length page")
+![Screenshot of financial support](financial-support.png "Financial support page")
+
+### 2. Adding in Markdown Guidance
+
+Some pages with text areas were missing the Markdown guidance, we added these in to the following pages:
+
+- About the Accredited Provider
+- Training with your organisation
+- Training with disabilities and other needs
+
+![Screenshot of accredited providers](accredited-providers.png "Accredited providers page")
+![Screenshot of about organisation](about-org.png "About organisation page")
+
+### 3. Removing Personal Qualities and Other Requirements
+
+Whilst originally we were looking to break these sections down into seperate pages, further analysis showed that the content in the personal qualities and other requirements fields were commonly repeated across all courses - making them less useful to candidates who are looking for things that make courses different from one another.
+
+We already surface information which is very similar to the content that providers are entering in personal qualities on the screen where candidates enter their personal statement. We feel that this is the most relevant place for this content in the journey.
```

Comment:
Can add a note about Other qualities and where we've relocated that information:

The content entered in the Other qualities field covered the mandatory checks before an offer of training can be accepted (DBS and fitness to train to teach checks). We already have this information on the start pages for Find and Apply and in the safeguarding declaration in Apply. We will add it to the email candidates receive when they get an offer to join a course, and to the Get Into Teaching website so that candidates can see this information throughout their journey.

User research with candidates also told us that some find it difficult to understand what a DBS check is, so we have explained what it means in the content.